.github/mergeable.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+mergeable:
+  - when: pull_request.*
+    name: "Changelog check"
+    validate:
+        - do: and
+          validate:
+            - do: dependent
+              changed:
+                file: 'src/**' 
+                required: ['CHANGELOG.md']
+            - do: dependent
+              changed:
+                file: 'deltachat-ffi/**'
+                required: ['CHANGELOG.md']
+    fail:
+      - do: checks
+        status: 'action_required' 
+        payload:
+          title: Changelog needs to be updated
```

Comment:
as it is only a warning that may be ignored when there are good reasons,, maybe better say sth. as "Changlong might need an update" resp. "Changelog not updated, please check if an update is needed."